src/init.cpp

```diff
@@ -1302,7 +1283,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
         // AMQP support is currently an experimental feature, so fail if user configured AMQP notifications
         // without enabling experimental features.
-        if (!fExperimentalMode) {
+        if (!GetBoolArg("-experimentalfeatures", false)) {
```

Comment:
I suggest reverting this and making `fExperimentalMode` accessible as a global.

Reply:
Oops lol, I'm suggesting the complete opposite to @str4d. Hmm, either way is fine I guess.

Reply:
Oh I see the problem. The argument parsing logic for AMQP isn't consistent with how the other experiential feature arguments are implemented. Yeah, leave this as it is for now.